### PR TITLE
Update number of expected exported tokenizers

### PR DESCRIPTION
**Pull request by IlyasMoutawwakil**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: tests/openvino/test_exporters_cli.py [hunk 1]
```diff
@@ -48,6 +48,7 @@
 from optimum.intel.utils.import_utils import (
     compare_versions,
     is_openvino_tokenizers_available,
+    is_tokenizers_version,
```
[the remark this answers is not in the document]
Comment (IlyasMoutawwakil) [reply tests/openvino/test_exporters_cli.py:51]: on it

[the remark this answers is not in the document]
Comment (echarlaix) [reply tests/openvino/test_exporters_cli.py:51]: also looks like the new release of `tokenizers` is incompatible with `openvino-tokenizers`, any uodated on this @eaidova @AlexKoff88 @apaniukov ?

[the remark this answers is not in the document]
Comment (apaniukov) [reply tests/openvino/test_exporters_cli.py:51]: I saw some issues with transformers 4.45/tokenizers 0.20 and fixed them on the master branch of openvino-tokenizers. You can use the nightly version for compatibility check, it updates daily:
```bash
pip install --pre -U openvino-tokenizers --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
```

File: tests/openvino/test_exporters_cli.py [hunk 2]
```diff
@@ -77,13 +78,13 @@ class OVCLIExportTestCase(unittest.TestCase):
         "t5": 0,  # no .model file in the repository
         "albert": 0,  # not supported yet
         "distilbert": 1,  # no detokenizer
-        "roberta": 2,
+        "roberta": 2 if is_tokenizers_version("<", "0.20") else 0,
```
Comment (apaniukov) [inline tests/openvino/test_exporters_cli.py:81]: It works with the current master of OV Tokenizers:

```bash
(venv) ~/python/openvino_tokenizers/t$ convert_tokenizer hf-internal-testing/tiny-random-roberta --with_detokenizer
Loading Huggingface Tokenizer...
Converting Huggingface Tokenizer to OpenVINO...
Saved OpenVINO Tokenizer: openvino_tokenizer.xml, openvino_tokenizer.bin
Saved OpenVINO Detokenizer: openvino_detokenizer.xml, openvino_detokenizer.bin
(venv) ~/python/openvino_tokenizers/t$ ls
openvino_detokenizer.bin  openvino_detokenizer.xml  openvino_tokenizer.bin  openvino_tokenizer.xml
(venv) ~/python/openvino_tokenizers/t$ pip list | grep tokenizers
openvino-tokenizers          2024.5.0.0 /home/apaniuko/python/openvino_tokenizers
tokenizers                   0.20.1
(venv) ~/python/openvino_tokenizers/t$ pip list | grep transformers
transformers                 4.45.2

```

**Pull request merged**